api/apps/services/dataset_api_service.py

```
@@ -220,6 +221,10 @@ async def delete_datasets(tenant_id: str, ids: list = None, delete_all: bool = F
     return True, {"success_count": success_count, "errors": errors[:5]}
 
 
+async def delete_datasets(tenant_id: str, ids: list = None, delete_all: bool = False):
```

Ruff (0.16.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)